java-manta-it/src/test/java/com/joyent/manta/http/TCPSocketConnectionTimeoutIT.java

```diff
@@ -64,11 +64,7 @@ public void verifyConnectionTimeoutSettingWorks() throws IOException {
 
         } catch(InterruptedIOException e){
             Assert.assertEquals(e.getClass().getSimpleName(),
-                    "ConnectTimeoutException");
-            Assert.assertTrue(e.getMessage().endsWith("connect timed out"),
-                    "ConnectTimeoutException didn't end with expected "
-                            + "connection timeout message. Actual exception:\n"
-                            + ExceptionUtils.getStackTrace(e));
+                    "ConnectTimeoutException", ExceptionUtils.getStackTrace(e));
```

Comment:
Someone who sees this test failure isn't going to understand why the stacktrace was printed. It would be better to provide a message like: "Expected exception ConnectTimeoutException was not thrown. Actual Exception: %s"